.travis.yml

```diff
@@ -14,12 +14,7 @@ php:
     - hhvm
 
 env:
-    global:
-        - TEST_COMMAND="composer test"
-    matrix:
-        - SYMFONY_VERSION=3.2.*
-        - SYMFONY_VERSION=3.1.*
-        - SYMFONY_VERSION=2.8.*
+    - SYMFONY_VERSION=2.8.* TEST_COMMAND="composer test"
```

Comment:
I think I would by default not enforce any Symfony version at all. And by the way, 2.8 should be handled fine by the job using the `--prefer-lowest` flag.